packages/cdktf/lib/backends/cloud-backend.ts

```diff
@@ -40,6 +40,13 @@ export class CloudBackend extends TerraformBackend {
     };
   }
 
+  public toMetadata() {
+    const cloud = [undefined, "app.terraform.io"].includes(this.props.hostname)
```

Comment:
Are you sure the value can only be exactly `app.terraform.io` and `undefined` here? What happens if `null` is passed or `https://app.terraform.io` or `app.terraform.io/`?

Reply:
Good call! I didn't check that..
Seems like they are invalid:
<img width="395" alt="image" src="https://user-images.githubusercontent.com/1112056/197995407-1c8894f0-e5dc-4fb3-b318-e52e22f55761.png">
<img width="368" alt="image" src="https://user-images.githubusercontent.com/1112056/197995509-6f223b3a-52bb-4655-a7af-4a9c4068a61e.png">

`app.terraform.io:80` is valid though 👀 Adding that one just in case, hehe.

Reply:
added ✅